public/index.html

```diff
@@ -401,5 +402,57 @@ <h3>Oups !</h3>
         </g>
       </svg>
     </div>
+    <script defer>
+      !function(t,e,n,s,a,c,i,o,p){t.AppsFlyerSdkObject=a,t.AF=t.AF||function(){(t.AF.q=t.AF.q||[]).push([Date.now()].concat(Array.prototype.slice.call(arguments)))},t.AF.id=t.AF.id||i,t.AF.plugins={},o=e.createElement(n),p=e.getElementsByTagName(n)[0],o.async=1,o.src="https://websdk.appsflyer.com?"+(c.length>0?"st="+c.split(",").sort().join(",")+"&":"")+(i.length>0?"af_id="+i:""),p.parentNode.insertBefore(o,p)}(window,document,"script",0,"AF","banners",{banners: {key: "%APPS_FLYER_WEB_KEY%"}});
+      AF('banners', 'showBanner', { bannerContainerQuery: '#smartBannerContainer' })
+    </script>
+    <script defer>
+      function waitForElementToExist(selector) {
+        return new Promise(resolve => {
+          if (document.querySelector(selector)) {
+            return resolve(document.querySelector(selector))
+          }
+
+          const observer = new MutationObserver(() => {
+            if (document.querySelector(selector)) {
+              resolve(document.querySelector(selector))
+              observer.disconnect()
```

Comment:
nittpick: l'observer n'est plus utilisé lorsqu'on resolve

```suggestion
              observer.disconnect()
              resolve(document.querySelector(selector))
```